boltdesignsystem/bolt
# Bolt List | Custom separator color

## Jira

http://vjira2:8080/browse/BDS-2027

## Summary

Add option to customize List separator color.

## Details

The default list separator color is very light. We want to allow users to customize the separator color for certain use cases, in particular a pipe-separated text list.

I updated the List CSS to include a CSS custom property named `--bolt-list-item-border-color` that can be used to customize the separator color.

## How to test

- Review files changed
- Compare [Landing Page on feature branch](https://feature-bolt-list-separator-opacity.boltdesignsystem.com/pattern-lab/patterns/03-blueprints-05-pages-t1-landing-pages-mission-landing--test-t1-mission-landing--test/03-blueprints-05-pages-t1-landing-pages-mission-landing--test-t1-mission-landing--test.html) to [Landing Page on master](https://master.boltdesignsystem.com/pattern-lab/patterns/03-blueprints-05-pages-t1-landing-pages-mission-landing--test-t1-mission-landing--test/03-blueprints-05-pages-t1-landing-pages-mission-landing--test-t1-mission-landing--test.html).
  - See the meta data ("60 Questions | 90 mins...") and verify that the separators appear stronger on the feature branch.
